patientsearch/src/js/helpers/utility.js

```diff
@@ -393,3 +404,28 @@ export function putPatientData(patientId, data, errorCallback, successCallback)
     if (successCallback) successCallback();
   });
 }
+
+export function addMamotoTracking(siteId, userId) {
+  if (document.querySelector("#matomoScript")) return;
```

Comment:
It isn't obvious to me where you're checking for a siteId of null (None) as would be the case if `MATOMO_SITE_ID` isn't defined?

Reply:
thanks @pbugni  for catching this, fixed.